build/build.ps1

```diff
@@ -227,6 +227,9 @@ if (!(Test-Path $CAKE_EXE)) {
     Throw "Could not find Cake.exe at $CAKE_EXE"
 }
 
+Write-Host "Restoring Nuget Packages..."
```

Comment:
Don't do this here. This script exist to bootstrap Cake. Cake is what orchestrates the build, not this script. Customize the build.cake script instead.